docs/zkapps/o1js/permissions.mdx

```diff
@@ -24,26 +24,27 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 # Permissions
 
 Permissions are an integral part of zkApp development because they determine who has the authority to interact and make changes to a specific part of a smart contract.
+
 Naturally, every smart contract must have proper permissions to prevent attacks or security holes.
 Permissions live on-chain, which means they are a part of the account representation on the network. Permissions are checked every time an account update tries to interact with an account.
 
 ## Types of Permissions
 
-There are 11 different types of permissions that a developer can access and adjust to guard a zkApp account:
+There are 11 different types of permissions that you can access and adjust to guard a zkApp account:
```

Comment:
```suggestion
There are 13 different types of permissions that you can access and adjust to guard a zkApp account:
```

Reply:
We reference 11 permissions but I count 13 here and also in o1js [here](https://github.com/o1-labs/o1js/blob/2bc9ee9d21a21e27cc7825683e22ebab050c9c9d/src/lib/account_update.ts#L166-L237)